crates/storage_models/src/enums.rs

```diff
@@ -638,6 +618,9 @@ pub enum PaymentIssuer {
     PentagonFederalCreditUnion,
     SynchronyBank,
     WellsFargo,
+    Paypal,
```

Comment:
Is the `PaymentIssuer` Type is `ENUM` or `String` in Database?

Reply:
String